pymc/distributions/multivariate.py

```diff
@@ -2218,6 +2218,174 @@ def logp(value, mu, W, alpha, tau):
         )
 
 
+class ICARRV(RandomVariable):
+    name = "icar"
+    ndim_supp = 1
+    ndims_params = [2, 1, 1, 0, 0, 0]
+    dtype = "floatX"
+    _print_name = ("ICAR", "\\operatorname{ICAR}")
+
+    def __call__(self, W, node1, node2, N, sigma, centering_strength, size=None, **kwargs):
+        return super().__call__(W, node1, node2, N, sigma, centering_strength, size=size, **kwargs)
```

Comment:
I think it'd be better to use a different word than "centering", since that's usually more about centered vs non-centered parameterizations.  Maybe "mean_zero_strength" or something similar instead?